collector/meminfo_darwin.go

```diff
@@ -0,0 +1,44 @@
+package collector
```

Comment:
This file is missing our header (we should have a header check script somewhere) and the typical `!no<collector>` build flag (which I think we can actually remove everywhere).